README.rst

```diff
@@ -394,6 +405,17 @@ Add ``openwisp_utils.loaders.DependencyLoader`` to ``TEMPLATES`` in your ``setti
         }
     ]
 
+5. Add swapper configurations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add the following to your ``settings.py``:
+
+.. code-block:: python
+
+    # Setting models for swapper module
+    DJANGO_X509_CA_MODEL = 'YOUR_MODULE_NAME.Ca'
+    DJANGO_X509_CERT_MODEL = 'YOUR_MODULE_NAME.Cert'
```

Comment:
can you do it like here please? https://github.com/openwisp/openwisp-firmware-upgrader/pull/60/files#diff-88b99bb28683bd5b7e3a204826ead112R396-R404